docs/admin/install.rst

```diff
@@ -142,7 +142,7 @@ The following dependencies have to be installed on the system:
 
 ``Git``
     https://git-scm.com/
-Pango, Cairo and related header files and gir introspection data
+Pango, Cairo and related header files and git introspection data
```

Comment:
gir stands for [GObject Introspection](https://en.wikipedia.org/wiki/GObject#GObject_Introspection) here:

```suggestion
Pango, Cairo and related header files and GObject introspection data
```